cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,21 @@
-/// <reference types="cypress" />
```

Comment:
There are no tests written for this code. Writing tests is important for ensuring the code works as expected and helps in catching any errors early.
Clean up it if you don't need this code

Reply:
Not fixed

Reply:
Not fixed

Reply:
Not fixed